crates/ty_server/src/session/options.rs

```diff
@@ -149,6 +155,7 @@ impl GlobalOptions {
             .experimental
             .map(|experimental| ExperimentalSettings {
                 rename: experimental.rename.unwrap_or(true),
```

Comment:
Oh shoot, the `unwrap_or` here was meant to be `false`. Can you make this change? Luckily, no one should be affected because there was only one setting in the experimental group.